build/azure-pipelines/build-common.yml

```diff
@@ -8,7 +8,16 @@ parameters:
 - name: ArtifactName
   type: string
 
-steps: 
+steps:
+- task: PowerShell@2

```

Comment:
instead of duplicating what's already in build-product.yml, make this value a parameter on the template and have build-product.yml pass in the value.

Reply:
the default value for the parameter can be $(Build.BuildNumber) just in case some pipeline that uses this yml doesn't set it. We only care about the tag for official builds

Reply:
Ack